signinapp/events.py

```diff
@@ -244,6 +256,8 @@ def edit_event():
         form.start.data = correct_time_for_storage(form.start.data)
         form.end.data = correct_time_for_storage(form.end.data)
         form.populate_obj(event)
+        event.cost = int(form.cost.data / 100)
```

Comment:
Each time you edit an event all of the money values are divided by 100. Seems like there's a missing multiplication somewhere here.